tools/distpartitioning/convert_partition.py

```diff
@@ -355,6 +356,33 @@ def _process_partition_gb(
     return indptr, indices[sorted_idx], edge_ids[sorted_idx]
 
 
+def update_node_map(node_map_val, end_ids_per_rank, id_ntypes, prev_last_id):
```

Comment:
copied from `partition.py` or modified accordingly? please add a comment to clarify it